roles/pbx/tasks/enable-or-disable.yml

```diff
@@ -84,7 +82,7 @@
       path: "{{ nginx_conf_dir }}/freepbx-nginx.conf"
       state: absent
 
-  when: not (pbx_use_nginx and pbx_enabled)
+  when: pbx_use_nginx and not pbx_enabled
```

Comment:
The original `when:` clause is what's needed.

As is the case throughout enable-or-disable.yml

Respectfully and with all due respect.

Reply:
Done